### ISSUE TITLE
CoF: always return a positive result for -launchapp or -nolaunchappwarn in 'hudCheckParm' engine function

### ISSUE THREAD

**Issue by SmileyAG** [YaLTeR/BunnymodXT#495]
Every time you launch the mod. version of Cry of Fear not through a launcher, but directly like regular GoldSrc mods, then every time a WinAPI window will pop up and asking you that it is better for you to use their launcher for *optimization*

But this is complete nonsense, code of this game itself is so terribly optimized, that getting stable 100 fps on half of maps is so rare and this is literally true for everyone who played
And besides, their launcher is broken on some configurations, and from the option there you can only set color depths, but what’s stopping us from doing this by using `-16bpp`, `-24bpp` or `-32bpp` in launch options?

Simply put, it’s completely useless to use their launcher; it’s better to skip this check by intercepting the `hudCheckParm` function in the engine and returning a positive result for `-launchapp` or `-nolaunchappwarn` option (yes, thank to them that they at least added options to ignore that pseudo-warning)
